Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
     "ui/app/utils/minijinja",
     "clients/rust",
     "clients/python-pyo3",
+    "provider-proxy",
```

Comment:
We're mixing `-` and `_`, we should pick one and fix all of the members here to match?

Reply:
I renamed `tensorzero_internal` to` tensorzero-internal` in https://github.com/tensorzero/tensorzero/pull/950